### PR TITLE
Blank Street Coffee

### PR DESCRIPTION
**Pull request by codeinabox**

### CODE REVIEW

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -378,6 +378,20 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Blank Street Coffee",
+      "locationSet": {
+        "include": ["gb-lon.geojson", "us"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:384]:
```suggestion
        "include": ["gb-lon.geojson", "us-ny.geojson"]
```
I notice that they mainly(or only?) have branch shop in New York, so I make it more detailed.

~~murmur: the map on their website looks awful~~

Comment (codeinabox) [reply data/brands/amenity/cafe.json]: [Their Twitter account](https://twitter.com/blankstreet) says they have locations in BK, MNY, BOS, DC & LDN that I took to mean Brooklyn, Manhattan, Boston, Washington and London

Comment (LaoshuBaby) [reply data/brands/amenity/cafe.json]: Yes, but on their website I only can find those shops(If you click "next" continuously, all of them are in NYC)

图片

[Their APP](https://apps.apple.com/us/app/blank-street/id1525240321):
图片

[Google map link](https://www.google.com/maps/d/embed?mid=1H4hjctjH6YxnkgqFN4vXXuYHTQa1_tVB&ll=40.72542756172712%2C-73.97802024999999&z=12) in website's footer(I use osm-everywhere plugin so google's tiles was replaced):
图片

If situation changed, we can change locationset angin.

Comment (codeinabox) [reply data/brands/amenity/cafe.json]: This article confirms that they have expanded into Boston and Washington https://dc.eater.com/2022/10/17/23402834/blank-street-coffee-nyc-dc-stores-openings

**Pull request merged**